content/03.methods.md

```diff
@@ -77,7 +77,8 @@ We required at least 60% of RNA-Seq reads mapped to the human reference or sampl
 #### SNV and INDEL calling
 
 We used four variant callers to call SNVs and INDELS from targeted DNA panel, WXS, and WGS data: Strelka2, Mutect2, Lancet, and VarDict.
-The same input interval BED files were used for both panel and WXS data.
+The same input interval BED files were used for both panel and WXS data and these were padded by 100 bp for all variant calling algorithm runs.
```

Comment:
Does this mean +100bp was added on both sides of the range? Or +50bp on each side? Also what was the reasoning behind this padding? Did this come from a recommendation from one of the callers?

Reply:
100bp on each side. This did come from recommendations from the callers, with the exception of lancet - we tested multiple and took the best for precision and performance.